src/renderer/entities/multisig/model/multisigs-model.ts

```diff
@@ -81,48 +81,56 @@ type GetMultisigsParams = {
   accounts: AnyAccount[];
 };
 
-const getMultisigsFx = createEffect(({ chains, accounts }: GetMultisigsParams): Promise<MultisigResult[]> => {
-  const requests = chains.flatMap((chain) => {
-    const multisigIndexer = networkUtils.getProxyExternalApi(chain);
+const getMultisigsFx = createEffect(({ chains, accounts }: GetMultisigsParams) => {
+  if (accounts.length === 0) return [];
 
-    if (nullable(multisigIndexer) || accounts.length === 0) return [];
+  const bindedRequestIdentities = scopeBind(requestIdentitiesFx, { safe: true });
+  const requests = chains.flatMap(async (chain) => {
```

Comment:
This looks reasonable given current urgency. For the future we should consider create a multi-chain msig indexer so we can fetch all the msigs in a single query